src/goto-instrument/cover_instrument_mcdc.cpp

```diff
@@ -14,6 +14,7 @@ Author: Daniel Kroening
 #include <langapi/language_util.h>
 
 #include <algorithm>
+#include <iterator>
```

Comment:
Procedural note: if this commit is moved to a separate PR, the CODEOWNERS-induced reviewer set will be a bit more reasonable.

Reply:
Yes, but PR dependencies don't really exist...

Reply:
I'm aware, but it could likely get reviewed and merged within minutes so that is in well before the build configuration.

Reply:
#2312